lua/ulx/modules/sh/cfc_brazil.lua

```diff
@@ -0,0 +1,141 @@
+local CATEGORY_TYPE = "Teleport"
+local MIN_X, MIN_Y, MIN_Z = -16384, -16384, -16384
+local MAX_X, MAX_Y, MAX_Z = 16384, 16384, 16384
+local HULL_BUFFER = 2
+local MAP_POSITION_DATA = {
+    ["gm_bigcity"] = {
+        centerZ = 14500 / 2 - 11400,
+        height = 14500 / 2,
```

Comment:
why are these numbers being divided by 2?

Reply:
centerZ and height (now renamed to playableHeight) are the heights and z-coordinate centers for the playable region of a map, basically the collection of all normally-accessible areas. The value divided by 2 is the height from top to bottom of this region, and the subtracted value is the bottom z-coordinate of the lowest accessible area. This results in finding the middle height of our playable region, and playableHeight is divided by 2 because if we go to centerZ + playableHeight without dividing by 2, we'll end up far above our playable zone. It's kinda like 14500 is a circumference and playableHeight is a radius.